MIGRATION.md

```diff
@@ -11,10 +11,27 @@ There are several ways to create a client connection in `cloudant-node-sdk`:
 
 ## Other differences
 1. Using the `dotenv` package to store credentials in a file is not recommended. See the [external file configuration section](https://github.com/IBM/cloudant-node-sdk#authentication-with-external-configuration) in our API docs for handling this feature in our new library.
-1. Fetching the database object first before performing additional operations is not required. For example, in the case of updating a document you would first call `getDocument` to fetch and then `putDocument` to update.
+1.  In `cloudant-node-sdk` all operations are performed from the scope of the client instance and
+    not associated with any sub-scope like the database. There is no need to instantiate a
+    database object to interact with documents - the database name is included as part of
+    document operations. For example, in the case of updating a document you would first call
+    `` getDocument({ db: `${dbName}`, docId:`${docId}`}) `` to fetch and then `` putDocument({ 
+    db: `${dbName}`, docId:`${docId}`}) `` to update. As a result of which the `use`
```

Comment:
This is just an alternative format suggestion.
```suggestion
    `getDocument({ db: ${dbName}, docId: ${docId}})` to fetch and then `putDocument({ 
    db: ${dbName}, docId: ${docId}})` to update. As a result of which the `use`
```

Reply:
6b34b7f where I removed also the `${..}`s